### DISCUSSION TITLE
Interfacing with NNLO Monte Carlo generators

### DISCUSSION THREAD

**Discussion by cschwan** [NNPDF/pineappl#62] [Ideas]
The following NNLO Monte Carlo are public, and we should investigate whether we could interface PineAPPL with them:

- [DYNNLO](https://www.physik.uzh.ch/en/groups/grazzini/research/Tools.html): DY
- [MATRIX](https://matrix.hepforge.org/): DY, W boson production, photon-pair production, ...
- [MCFM](https://mcfm.fnal.gov/): Z pt, ...
- [top++](http://www.precision.hep.phy.cam.ac.uk/top-plus-plus/): top-pair production
- [Vrap](https://www.slac.stanford.edu/~lance/Vrap/): DY rapidity, also for fixed-target DY processes with the patches in https://github.com/NNPDF/external/tree/master/vrap/vrap-0.9, see also
  - https://github.com/NNPDF/external/pull/58 and
  - https://github.com/NNPDF/external/pull/59
- [Hightea](https://www.precision.hep.phy.cam.ac.uk/hightea/): many LHC processes

**Comment by enocera** [comment]
Just to mention that:
1) top++ only does total cross sections; differential top pair distributions can usually be taken from NNLO FastNLO grids here https://www.precision.hep.phy.cam.ac.uk/results/ttbar-fastnlo/;
2) there are NNLO FastNLO tables (mostly for DIS+jets) here https://ploughshare.web.cern.ch/ploughshare/download.php.

**Comment by Zaharid** [comment]
I suppose top++ will be superseded by hightea.

**Reply by cschwan** [reply]
I added it to the list!
